helm/wts/templates/external-secret.yaml

```diff
@@ -1,4 +1,26 @@
-{{- if .Values.global.externalSecrets.deploy }}
+---
+{{ if .Values.global.externalSecrets.deploy }}
+{{ if not .Values.externalSecrets.createWtsOidcClientSecret }}
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: wts-oidc-client
+spec:
+  refreshInterval: 5m
+  secretStoreRef:
+    name: {{include "common.SecretStore" .}}
+    kind: SecretStore
+  target:
+    name: wts-oidc-client
+    creationPolicy: Owner
+  dataFrom:
+  - extract:
+      key: wts-oidc-client
```

Comment:
I think we need to parameterize this, because different environments will have different secrets. Either way, I think having the option is a good idea.